File: TP1 java/java fede/Ejercicio 3/Estrenos.java
import java.util.ArrayList;

public class Estrenos {

    private String titulo;
    private ArrayList<Serie> series;

    public Estrenos(String titulo){
        this.titulo = titulo;
        this.series = new ArrayList<>();
    }

    public void agregarSerie(Serie ser){
        this.series.add(ser);
    }

    public void imprimir(){
        System.out.println(this.titulo);
        for(Serie ser:this.series){
            ser.imprimirSer();
        }
    }
}

File: TP1/TP1 python/python fede/ejercicio 3/Estrenos.py
class Estrenos:
    
    def __init__(self, titulo):
        self.titulo = titulo
        self.series = []
        
    def agregarSerie(self, serie):
        self.series.append(serie)
        
    def imprimir(self):
        for ser in self.series:
            ser.imprimirSerie()